app/assets/javascripts/views/export_csv_view.js

```diff
@@ -0,0 +1,53 @@
+// Handles getting info about bulk media thresholds
+ELMO.Views.ExportCsvView = class ExportCsvView extends ELMO.Views.ApplicationView {
+  get events() {
+    return {
+      'click #response_csv_export_options_download_media': 'calculateMediaSize'
+    };
+  }
+
+  initialize(params) {
+    $(".calculating-info").hide();
+    $(".error-info").hide();
+    $(".media-info").hide();
+  }
+
+  calculateMediaSize(event) {
+    if (($(event.target)).is(':checked')) {
+      $("input[type=submit]").prop("disabled", true);
+      this.spaceLeft();
+      $(".media-info").show();
+
+    } else {
+      $("input[type=submit]").removeAttr("disabled");
+      $(".media-info").hide();
+      $(".error-info").hide();
+    }
+  }
+
+  spaceLeft() {
+    $(".calculating-info").show();
+
+    $.ajax({
```

Comment:
In JS, requests like this happen asynchronously by default, so you should use `async/await` if you want to make sure the call finishes before continuing code execution.

I'm not sure if you've used this much in the past, so with a deadline to deploy this week I went ahead and added it -- you can see in https://github.com/thecartercenter/nemo/pull/850/commits/9fb1f0fa89c7f11ac1037418ba2118ad93d035d0, lemme know if you wanna talk more!